content/04. box_1_experimental_design.md

```diff
@@ -1,37 +1,36 @@
-Ensuring absence of any overlap or relatedness among data points or samples (e.g., samples with familial relationship, samples from same patient) used in the training set and evaluation set is important to avoid data leakage during model training.
-Specifically in cases of rare genetic diseases where many samples can contain familial relationships, special care should be taken while crafting the training and testing sets to ensure that no data leakage occurs and the trained model has high generalizability.
+While the evaluation dataset helps us refine a model's "knowledge" of patterns in the training data, the held-out validation set helps us test the generalizability of the model.
 
+If a model is generalizable, it is able to make accurate predictions on new data.
+High generalizability of a model on previously unseen data suggests that the model has identified important patterns in the data that are not unique to the data used for training and tuning.
+Generalizability can be affected if _data leakage_ occurs during training of the model, i.e., if a model is exposed to the same or related data points in both the training set and the held-out test set.
+Ensuring absence of any overlap or relatedness among data points or samples used in the training set and evaluation set is important to avoid data leakage during model training.
+Specifically, in cases of rare genetic diseases where, for example, many samples can contain familial relationships or data from the same patient could be collected by multiple specialists at different clinical facilities, special care should be taken while crafting the training and testing sets to ensure that no data leakage occurs and the trained model has high generalizability.
 
 ### Training and testing
 
-The implementation of a machine learning experiment begins with splitting a single dataset of interest such that 90% of the data is used for training (generally subdivided into the _training dataset_ and the _evaluation dataset_), and remaining 10% of the data is used for testing or validation (as the _held-out validation dataset_).
-This makes sure that all the datasets involved in training and testing a model maintain uniformity in the features.
-In case of rare diseases where multiple datasets may be combined to make a large enough training dataset, special care should be taken to standardize the features and the patterns therein.
-The iterative training stage helps the model learn important patterns in the training dataset and then use the evaluation dataset to test for errors in prediction and update its learning parameters (_hyperparameter tuning_).
+The implementation of a ML experiment begins with splitting a single dataset of interest such that 90% of the data is used for training (generally subdivided into the _training dataset_ and the _evaluation dataset_), and remaining 10% of the data is used for testing or validation (as the _held-out validation dataset_).
```

Comment:
This is on my list for my "deeper look at the references pass-through" but what do you think about toning down how definitive this is about the 90/10 split?

Reply:
I at least 90% agree with this. But maybe 75%. Or maybe 95%. :)

```suggestion
The implementation of a ML experiment begins with splitting a single dataset of interest such that a large proportion of the of the data (e.g., 70-90%) is used for training (generally subdivided into the _training dataset_ and the _evaluation dataset_), and the remaining data is used for testing or validation (as the _held-out validation dataset_).
```